### ISSUE TITLE
Convergence of forward model

### ISSUE THREAD

**Issue by bkumardevan07** [as-ideas/TransformerTTS#101]
Hi,
While training forward model I observed that my validation loss is rising at just after 20k steps? Is there something wrong going on? How well is your pretrained model?

**Comment by cfrancesco** [comment]
This is something I always experienced when training forward models. Durations probably easily overfit. Practically has not been an issue, despite this making harder to understand the status of training of course.
